openedx/core/djangolib/blockstore_cache.py

```diff
@@ -32,7 +33,7 @@
 # (Note that we do usually explicitly invalidate this cache during write
 # operations though, so this setting mostly affects actions by external systems
 # on Blockstore or bugs where we left out the cache invalidation step.)
-MAX_BLOCKSTORE_CACHE_DELAY = 60 * 5
+MAX_BLOCKSTORE_CACHE_DELAY = 60
```

Comment:
For anyone watching, this is probably the issue -- we started holding onto the cache value too long.

Reply:
Scratch that. It looks like the issue has more to do with memcached.